package.json

```diff
@@ -79,6 +78,7 @@
     "prettier:format": "prettier --write \"{,**/}*.{js,ts,cjs,mjs,json,md,yml,java}\"",
     "pretest": "npm run lint",
     "test": "mocha test generators --no-insight --forbid-only --parallel --max-old-space-size=4096",
+    "test-single": "mocha test test/cli/cli.spec.js",
```

Comment:
Is it important?
```suggestion
```

to test a single file just run
```
npx mocha  test/...
```

Reply:
No, I added it by mistake. It's removed.